### PR TITLE
Fix table delimiter [0.1]

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

File: src/main/scala/is/hail/utils/TextTableReader.scala [hunk 1]
```diff
@@ -12,15 +12,23 @@ import org.apache.spark.sql.Row
...
+    val matchSep: Int => Int = separator.length match {
+      case 0 => fatal("Hail does not currently support 0-character separators")
+      case 1 =>
+        val sepChar = separator(0)
+        (i: Int) => if (s(i) == sepChar) 1 else -1
+      case _ =>
+        val p = Pattern.compile(separator)
+        val m = p.matcher(s)
+
+      { (i: Int) =>
```
Comment (cseed) [inline src/main/scala/is/hail/utils/TextTableReader.scala]: Same indent problem?  What's up with that, IntelliJ?

Comment (tpoterba) [reply src/main/scala/is/hail/utils/TextTableReader.scala]: wait, this looks right. No?

Comment (tpoterba) [reply src/main/scala/is/hail/utils/TextTableReader.scala]: oh, no it doesn't. wtf.

Comment (tpoterba) [reply src/main/scala/is/hail/utils/TextTableReader.scala]: must be an intelliJ bug. Fixed by moving the brace.

**Pull request merged**